nautobot/extras/migrations/0056_alter_configcontext_roles.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2022-12-12 14:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("extras", "0055_collect_roles_from_related_apps_roles"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="configcontext",
+            name="roles",
+            field=models.ManyToManyField(blank=True, related_name="_extras_configcontext_roles_+", to="extras.Role"),
```

Comment:
Where do existing ConfigContexts that reference `DeviceRole` instances get migrated to instead reference the new `Role` instances?